product_internal_reference_generator/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Product Internal Reference Generator",
+    "summary": """Product template and variant reference based on sequence""",
+    "author": "Ilyas, Ooops, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/product-variant",
+    "category": "Sale",
+    "version": "14.0.1.0.0",
+    "depends": ["stock"],
+    "data": [
+        "security/groups.xml",
+        "security/ir.model.access.csv",
+        "views/product.xml",
+    ],
+    "demo": ["demo/demo.xml"],
```

Comment:
**chore**: Please give files meaningful names according to the guidelines

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming